moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -40,7 +40,9 @@ move_group_name:  panda_arm  # Often 'manipulator' or 'arm'
 planning_frame: panda_link0  # The MoveIt planning frame. Often 'base_link' or 'world'
 
 ## Other frames
-ee_frame: panda_link8  # The name of the end effector link, used to return the EE pose
+ee_frame: panda_hand  # The name of the end effector link, used to return the EE pose
+
+command_frame_is_stationary: true  # Twist commands are as seen from a stationary frame.
```

Comment:
I'd omit the question mark. Maybe:

```suggestion
# If true, twist commands are given in a stationary frame; otherwise, they are in a moving frame.
# See https://github.com/ros-planning/moveit2/issues/2150 for more details
command_frame_is_stationary: true
````

Reply:
^sounds good